File: mario-less/mario.c

#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int i , j , height;
     while (true) {
                height = get_int("Height: ");

                if (height <= 8 && height > 0) {
                break;
                  }
                 }

    for(i=1; i < height+1; i++) {


        for(j=i; j<height; j++) {

        printf(" ");


        }
        for(j=0 ; j<i; j++) {

             printf("#");

         }
         printf("\n");

    }


}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
while True:
    n = get_int("Height:")
    if n > 0 and n <= 8:
        break

for i in range (1, n+1):
    for j in range (i, n):
        print(" ", end = "")
    for j in range (i):
        print("#", end = "")
    print("  ", end = "")
    for j in range (i):
         print("#", end = "")
    print()
